docs/charts/radar.md

```diff
@@ -143,7 +143,11 @@ The interaction with each point can be controlled with the following properties:
 
 ## Configuration Options
 
-Unlike other charts, the radar chart has no chart specific options.
+The radar chart defines the following configuration option. This option is merged with the global chart configuration option, `Chart.defaults.global`, to form the option passed to the chart.
```

Comment:
This sounds slightly off to me.

> "global chart configuration option, `Chart.defaults.global`"

There's multiple options defined in `Chart.defaults.global`. I would make "option" plural especially in this sentence, but probably everywhere in the new text

Reply:
Fixed, thanks.